tests/create_and_delete_test.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+from unittest.mock import patch
+from pieces.commands import create_asset, delete_asset,startup
+from pieces.commands import commands_functions
+from pieces.api.assets import get_asset_by_id
+
+class TestAssetFunctions(unittest.TestCase):
+
+    @patch('builtins.input', side_effect=['y', 'y'])
+    @patch('pyperclip.paste', return_value='print("Hello, World!")')
+    def test_create_and_delete_asset(self, mock_paste, mock_input):
+        startup()
+        # Call create_asset and store the returned asset
+        create_asset()
+
+        # Ensure that it is correctly assigned to the current asset
+        new_asset=list(commands_functions.current_asset)[0]
+
+
+        asset_object = get_asset_by_id(new_asset)
+
+        get_asset_by_id(new_asset) # Checks that the asset was created with no errors
```

Comment:
I'm guessing this will throw an error if this is not proper? therefore failing the test?

It might be nice to return the asset from the `create_asset` function, then assert that the asset's id is present in the list

Reply:
yes this should be tagged as an issue currently any error is handled and output the error to the user in a red color so that should be marked as an issue